spec/clear_spec.lua

```diff
@@ -0,0 +1,30 @@
+-- spec/clear_spec.lua
+local clear = require "terminal.clear"
+
+describe("Clear Module Tests", function()
+
+    it("should return correct ANSI sequence for clearing entire screen", function()
```

Comment:
it looks like your whitespace settings deviate from the rest of the repo. Can we keep this in line please?

if using VSCode, try this plugin: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

Also check the vertical whitespace in other test files;
- 2 lines between tests
- 3 between major code blocks (describes, setup - it blocks, etc)